lib/xnvme_be_spdk_dev.c

```diff
@@ -233,15 +194,18 @@ _spdk_env_init(struct spdk_env_opts *opts)
 
 	xnvme_spdk_choker_on();
 
-	if (!opts) {
-		opts = &_spdk_opts;
+	spdk_env_opts_init(&env_opts);
+	env_opts.name = "xnvme";
+	env_opts.shm_id = 0;
```

Comment:
This line should be removed, spdk_env_opts_init sets this to -1 which will configure dpdk with the flag --no-shconf. using shm_id=0 will setup multiprocess environment, you may need to move it to inside the if block